app/third_party/weather_api.rb

```diff
@@ -0,0 +1,10 @@
+class WeatherApi
+  BASE = 'http://api.openweathermap.org/data/2.5'
+  KEY = '712h3jnf89sk0a87gbnma9'
```

Comment:
WHY ARE IS THIS HERE?!?!?!?!??!?!?!?!?! CREATE A SECRET AND ADD IT TO YOUR ENV 
![image](https://user-images.githubusercontent.com/11698687/43471070-7c80d304-94c0-11e8-8877-611efd433767.png)